ansible_collections/arista/avd/docs/pyavd.md

```diff
@@ -112,4 +110,7 @@ md-toc>=8.1.8
 ::: pyavd.validation_result
     options:
       show_root_toc_entry: false
-      paths: ../../../../python-avd
+
+::: pool_manager.avdpoolmanager
```

Comment:
I am not sure if you are talking about the docs and md file your are commenting on, or if this is a more general question.
1. For docs we have to keep it under the collection, since that is the mkdocs root.
2. For code we have to keep it together withing the ansible collection for it to be shipped as part of the collection. Remember that pyavd is not a dependency of ansible-avd (yet)

Reply:
no longer relevant